k8s/upgrades/0.8.1-0.8.2/jiva_volume_upgrade.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+target_upgrade_version="0.8.2"
+current_version="0.8.1"
+
+function usage() {
+    echo
+    echo "Usage:"
+    echo
+    echo "$0 <pv-name>"
+    echo
+    echo "  <pv-name> Get the PV name using: kubectl get pv"
+    exit 1
+}
+
+function setDeploymentRecreateStrategy() {
+    dns=$1 # deployment namespace
+    dn=$2  # deployment name
+    currStrategy=`kubectl get deploy -n $dns $dn -o jsonpath="{.spec.strategy.type}"`
+    rc=$?; if [ $rc -ne 0 ]; then echo "Failed to get the deployment stratergy for $dn | Exit code: $rc"; exit; fi
+
+    if [ $currStrategy != "Recreate" ]; then
+       kubectl patch deployment --namespace $dns --type json $dn -p "$(cat patch-strategy-recreate.json)"
+       rc=$?; if [ $rc -ne 0 ]; then echo "Failed to patch the deployment $dn | Exit code: $rc"; exit; fi
+       echo "Deployment upgrade strategy set as recreate"
+    else
+       echo "Deployment upgrade strategy was already set as recreate"
+    fi
+}
+
+if [ "$#" -ne 1 ]; then
+    usage
+fi
+
+pv=$1
+replica_node_label="openebs-jiva"
+
+source snapshotdata_upgrade.sh
```

Comment:
lets pick snapshotdata_upgrade.sh file as well

Reply:
and, we need to add kn cstor_volume_upgrade script as well

Reply:
done